src/components/organisms/dashboard/projectsTable.tsx

```diff
@@ -73,12 +78,41 @@ export const DashboardProjectsTable = () => {
 				...stats.sessionCounts,
 				status: projectStatus,
 				lastDeployed,
+				deploymentId,
 			};
 		}
 		setProjectsStats(Object.values(projectsStats));
 		setIsLoading(false);
 	};
 
+	const handleDeploymentStop = useCallback(
+		async (deploymentId: string) => {
+			const { error } = await DeploymentsService.deactivate(deploymentId);
+
+			if (error) {
+				addToast({
+					message: (error as Error).message,
+					type: "error",
+				});
+
+				return;
+			}
+
+			addToast({
+				message: tDeployments("actions.deploymentDeactivatedSuccessfully"),
+				type: "success",
+			});
+			LoggerService.info(
+				namespaces.ui.deployments,
+				tDeployments("actions.deploymentDeactivatedSuccessfullyExtended", { deploymentId })
+			);
+
+			await loadProjectsData(projectsList);
```

Comment:
If the deactivate deployment succeeds, we don't need to reload all deployments for all projects, but only for this one.